packages/dapp/src/components/OnRamper/OnRamper.tsx

```diff
@@ -0,0 +1,58 @@
+import { useTheme } from '@mui/material';
+import { OnRamperIFrame } from './index';
+
+function removeHash(str) {
+  if (str.startsWith('#')) {
+    return str.substring(1);
+  }
+  return str;
+}
+
+export const OnRamper = () => {
+  const theme = useTheme();
+
+  const onRamperConfig = {
+    defaultCrypto: 'ETH',
```

Comment:
Something still doesn´t seem right with the defaultCrypto. Further down, on where this is used I run into the following issue - it returns AAVE (probably because it is not correctly parsed). Whenever I move this "defaultCrypto" as the first parameter to be passed in the query-string, it works like a charne - but then the themeMode does not apply correctly.
Seems, as those have to be first in line? Idk, I left it as it is, having AAVE as a default for now.